### DISCUSSION TITLE
Thetis Security Key

### DISCUSSION THREAD

**Discussion by wyzemoro** [keepassxreboot/keepassxc#9396] [Q&A] [answered]
Is there a plan to support Thetis security key?

**Comment by droidmonkey** [comment, marked as answer]
No, there are no plans

**Comment by ashleysommer** [comment]
Hi @wyzemoro 
There is an update to this answer. 
There is work progressing on implementing support for FIDO2 security keys in KeepassXC. Once that work is complete and merged, you will be able to use your Thetis security key. https://github.com/keepassxreboot/keepassxc/discussions/9506
